<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtSightnessClass</class>
 <widget class="QMainWindow" name="QtSightnessClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1137</width>
    <height>679</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtSightness</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>30</y>
      <width>521</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>3ds</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>基于人脸识别和瞳孔检测的亮度调节装置</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_begin_simulation">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>490</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>开始自适应调节</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>1081</width>
      <height>331</height>
     </rect>
    </property>
    <widget class="QSlider" name="verticalSlider_color">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>30</y>
       <width>22</width>
       <height>241</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QSlider" name="verticalSlider_brightness">
     <property name="geometry">
      <rect>
       <x>880</x>
       <y>30</y>
       <width>22</width>
       <height>241</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>300</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>色温</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>880</x>
       <y>300</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>亮度</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox">
     <property name="geometry">
      <rect>
       <x>740</x>
       <y>270</y>
       <width>46</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Adobe 黑体 Std R</family>
      </font>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_2">
     <property name="geometry">
      <rect>
       <x>870</x>
       <y>270</y>
       <width>46</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Adobe 黑体 Std R</family>
      </font>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_video">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>171</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>3ds</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>图像显示区</string>
    </property>
   </widget>
   <widget class="QTimeEdit" name="timeEdit">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>580</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>3ds</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1137</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>退出</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>帮助</string>
    </property>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QtSightness.qrc"/>
 </resources>
 <connections/>
</ui>
